package.json

```diff
@@ -153,5 +153,33 @@
       "path": "./dist/grommet.min.js",
       "maxSize": "110 kB"
     }
+  ],
+  "keywords": [
+    "grommet",
+    "grommet.io",
+    "component",
+    "library",
+    "design system",
+    "framework",
+    "react",
+    "theming",
+    "templates",
+    "sticker sheets",
+    "design patterns",
+    "app templates",
+    "atomic design",
+    "accessibility",
+    "WCAG 2.1",
+    "keyboard navigation",
+    "screen reader tags",
+    "color",
+    "type",
+    "layout",
+    "typgraphy",
+    "interaction",
+    "ui",
+    "user interface",
+    "gui",
+    "graphical user interface",
```

Comment:
I think this trailing comma needs to be removed.

Reply:
I'm pretty new to Git... Or rather, I've found my way back after A LONG time gone. I don't really know how to do the changes of your request correctly within this PR. But it seems like you fixed it by yourself? :)

Reply:
nope not fixed. you can basically just do another commit fixing it and push it in your branch. it will be picked up automatically